### DISCUSSION TITLE
Feature request: Rounded corners on artboards

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#520] [Ideas]
![image](https://user-images.githubusercontent.com/11275573/106600287-7c314d80-655a-11eb-832b-077ce677b455.png)

Figma:
![image](https://user-images.githubusercontent.com/11275573/106600048-32e0fe00-655a-11eb-9f41-db49b5014fc5.png)

**Comment by jdittrich** [comment]
Would that mean "rounded corners should be possible" or "rounded corners as default"?
I personally think it is nice if it works, but I would also be fine with having a rounded-corner rectangle on my artboards.

**Reply by Manueljlin** [reply]
I meant the former

**Reply by myfunnyandy** [reply]
Hi @Manueljlin, thanks for this proposal, and thanks @jdittrich por helping to clarify it. We will consider to add this property to artboards.

**Comment by myfunnyandy** [comment]
[Taiga User Story](https://tree.taiga.io/project/penpot/us/411)
